### PR TITLE
🔖 feat: add beta release for the notion-proxy-rest-api

### PR DESCRIPTION
**Pull request by gengjozsef**
- add version in dandelion.json

### CODE REVIEW

File: dandelion.json [hunk 1]
```diff
@@ -635,6 +635,10 @@
       "path": "bundles/no-payment-credit-memo/",
       "version": "1.0.0"
     },
+    "notion-proxy-rest-api": {
+      "path": "bundles/notion-proxy-rest-api",
+      "version": "1.0.0.beta.1"
```
Comment (SpiderDan98) [inline dandelion.json]: https://semver.org/
```suggestion
      "version": "1.0.0-beta.1"
```

**Review by SpiderDan98 — APPROVED** [review]

File: dandelion.json (hunk 1 above)
Comment (julianhyatt) [inline dandelion.json]: -beta

**Review by julianhyatt — APPROVED** [review]

File: dandelion.json (hunk 1 above)
Comment (gengjozsef) [reply dandelion.json]: @thx copy paste error from:
 "stock-api": {
      "path": "bundles/stock-api",
      "version": "1.0.0.beta.0"
    },

it is resolved

Comment (gengjozsef) [reply dandelion.json]: @julianzimmermann thnx copy paste error from: 
@thx copy paste error from:
 "stock-api": {
      "path": "bundles/stock-api",
      "version": "1.0.0.beta.0"
    },

resolved

**Pull request merged**